.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## Description, Motivation and Context
+
+Please include a summary of the change or what problem does it solve. Please also include relevant motivation and context.
+
+<!--- If it relates to an open issue, please link to the issue here.
+e.g.
+[issue 414](https://github.com/replicatedhq/troubleshoot.sh/issues/414)
+-->
+
+## Checklist
+
+- [ ] The impact of the change checked.
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] The commit message(s) follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
+- [ ] Documentation added in /docs or commit messages for bug fixes / features
```

Comment:
Docs are stored in the troubleshoot.sh repo - the /docs and /design directories in this repo are for design work only